cmd/root.go

```diff
@@ -134,6 +140,13 @@ func Run(spec *Specification, hal hal.InBand) (*event.EventEmitter, error) {
 		return eventEmitter, err
 	}
 
+	grpcClient, err := NewGrpcClient(certsClient, eventEmitter)
+	if err != nil {
+		log.Error("failed to fetch GRPC certificates", "error", err)
+		return eventEmitter, err
+	}
+	hammer.GrpcClient = grpcClient
```

Comment:
grpcclient is required for UpdateBmcSuperuserPassword, so this is too late

Reply:
good catch :+1: